lib/mappings/bib-mapping.json

```diff
@@ -6,10 +6,13 @@
       }
     ]
   },
-  "aeonURL": {
```

Comment:
We're not removing the `aeonURL` property, but we don't rely on this mapping file to build it. Just cleaning up.